staff/almu-itur/cv/index.html

```diff
@@ -0,0 +1,78 @@
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <header>
+            <h1>ALMUDENA ITURGOYEN</h1>
+            <h2>Aspiring developer</h2>
+            <h6>Email</h6><h6>Phone</h6>
+    </header>
+        <div class="summary">
+            <h3>Summary</h3>
+            <p>Versatile lawyer with over 7 years of working experience in international litigation, financial claims and compliance. Entrepreneurial mindset, learning addict and always up for a challenge.</p>
+        </div>
+
+        <div class="experience">
+            <h3>Experience</h3>
+                <h4>Founder @ Olea Legal</h4>
+                <h5>October 2016 - Current</h5><h5>London</h5>
+                <ul>
+                    <li>Reviewing whitepaper of ICO projects in compliance with the regulations where the tokens are issued.</li>
+                    <li>Advising European clients on different aspects of personal injury and contractual claims connected to Spain such as limitation, liability, prospects, causation, case-law, quantum and others.</li>
+                    <li>Acting for insurance companies in international claims connected to Spain dealing with all aspects of the claim.</li>
+                    <li>Reviewing and drafting contracts.</li>
+                    <li>Translating documents from English into Spanish.</li>
+                </ul>
+
+                <h4>Lawyer @ Olea Legal</h4>
+                <h5>Mayo 2016 - December 2017</h5><h5>London</h5>
+                <ul>
+                    <li>Advising and acting for insurance companies and individuals in international personal injury claims arising from road traffic accidents, sports accidents, falls, occupiers’ liability, defective products, breach of contract and criminal acts.</li>
+                    <li>Specialisation: international law (contractual and non-contractual). Sector: insurance.</li>                          • Sector: insurance.</li>
+                </ul>
+        </div>
+       
+        <div class="education">
+            <h3>Education</h3>
+                <ul>
+                    <li><span>Skylab Bootcamp</span>Skylab Coding Academy Barcelona (current)</li>
+                    <li><span>Masters Degree – International Law</span>ISDE & Universidad of Barcelona (2010-2011)</li>
+                    <li><span>University Degree - Law</span>Pompeu Fabra University, Barcelona (2006-2010)</li>
+                    <li><span>Degree – Telecommunications engineering (2 courses)</span>Ramon Llull University, Barcelona (2004-2006)</li>
+                </ul>
+
+        </div>
+
+        <div class="itskills">
+           <ul>
+               <li>Javascript</li>
+               <li>HTML</li>
+               <li>CSS</li>
+               <li>C</li>
+
+           </ul>
+        </div>
+        
+        <div class="languages">
+            <ul>
+                <li><span>Spanish</span>: Native</li>
```

Comment:
Why do you use span tags? For future styling/javascript functionality?

Reply:
I used it to highlight content but I will use strong, instead